pkg/tagger/tagger.go

```diff
@@ -327,3 +329,18 @@ func copyArray(source []string) []string {
 	copy(copied, source)
 	return copied
 }
+
+// tagCardinalityToString returns a string representation of a TagCardinality
+// value.
+func tagCardinalityToString(c collectors.TagCardinality) string {
```

Comment:
we have `stringToTagCardinality` in `pkg/tagger/global.go`, I think we can move both functions in a new `pkg/tagger/utils.go`,  declare the strings `high` `low` `orchestrator` as const and share them between the two functions